packages/scaffold-utils/src/SolanaHelpersUtils.ts

```diff
@@ -0,0 +1,71 @@
+import { PresetsUtil } from './PresetsUtil.js'
+
+import type { CaipNetwork } from '@web3modal/core'
+import type { Chain, Provider } from './SolanaTypesUtil.js'
+
+export const SolHelpersUtil = {
+  getChain(chains: Chain[], chainId: string | null) {
+    const chain = chains.find(lChain => lChain.chainId === chainId)
+    if (chain) {
+      return chain
+    }
+
+    return chains[0]
```

Comment:
wouldn't this cause unexpected behaviors? should we maybe have a hardcoded default chain?